### PR TITLE
Fix/make reservation work for opentable

### PR DESCRIPTION
**Pull request by reicolee**

### CODE REVIEW

File: templates/_reservations.html [hunk 1]
```diff
@@ -5,17 +5,17 @@
 
   <div class="modal-dialog">
     <div class="modal-content left">
-        <form method="get" id="TU_form" target="_blank" action="https://www.opentable.com/restaurant-search.aspx">
+        <form method="get" id="TU_form" action="https://www.opentable.com/restaurant-search.aspx">
 
-        <input type="hidden" name="RestaurantID" id="RestaurantID" value="254077">
+        <input type="hidden" name="RestaurantID" id="RestaurantID" value="{{ account.services.opentable.id }}"> <!-- 254077 --> <!-- 186550 -->
```
Comment (EricFries) [inline templates/_reservations.html]: Can you delete these hard coded values that are commented out?

File: templates/_reservations.html [hunk 2]
```diff
@@ -5,17 +5,17 @@
 
   <div class="modal-dialog">
     <div class="modal-content left">
-        <form method="get" id="TU_form" target="_blank" action="https://www.opentable.com/restaurant-search.aspx">
```
Comment (EricFries) [inline templates/_reservations.html:8]: Check if there is any CSS using the id `TU_form`.  If not, delete that from the form since we're not using it.

Comment (reicolee) [reply templates/_reservations.html:8]: already deleted both the id and the hardcoded id from the html, can check and merge if everything is okay.

**Comment by EricFries** [comment]
Looks good.  Let's merge this into master and push.  Let me know if you need help doing that.

**Pull request merged**